### ISSUE TITLE
Pinned dependency for `aiohttp-client-cache` causes conflicts with `homeassistant` package

### ISSUE THREAD

**Issue by utkashd** [GrandMoff100/HomeAssistantAPI#177]
# Problem

This package has a dependency `aiohttp-client-cache` which has a sub-dependency `attrs`. `aiohttp-client-cache` in this package is pinned to an old version that caps `attrs` to `< 22.x`:
https://github.com/GrandMoff100/HomeAssistantAPI/blob/bd0e7af34564ca26adda07723430b1c07aa4a0aa/pyproject.toml#L20

This unfortunately conflicts with the latest `homeassistant` package and some other notable packages. (`homeassistant` currently requires `attrs == 23.2.0`)

----
# Potential Fix

I was able to unblock myself with this modification, and I imagine nothing breaks as a result of it:

```toml
aiohttp-client-cache = ">=0.6.1"
```

With aiohttp-client-cache 0.11.0 installed, this succeeds:

```python
from homeassistant_api import Client

client = Client(api_url='https://homeassistant.local:8123/api', token=os.environ.get('HA_TOKEN'), use_async=True)

await client.async_get_entities()
```

**Comment by GrandMoff100** [comment]
Yeah, it probably does not need to be pinned to that specific version. I think poetry auto pinned it to the latest version at the time. I'll run the tests with the newer one and update the pinned dependency version if they all pass.

**Comment by GrandMoff100** [comment]
@utkashd sorry for the delay, but a new version that is compatible with `attrs==23.2.0` is not published (`4.2.2.post1`)

Happy coding

**Comment by utkashd** [comment]
No rush at all on my end, I'll aim to contribute instead of opening an issue next time. Thanks so much for the quick fix!!
